maven-caching-extension/src/site/markdown/CACHE-GETTING-STARTED.md

```diff
@@ -0,0 +1,74 @@
+<!---
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+## Getting Started
+
+To on-board incremental Maven you need to complete several steps:
+
+* Declare caching extension in your project
+* Add cache config in `.mvn` (optional) to customize default behavior
+* Validate build results and iteratively adjust config to properly reflect project specifics
+* Setup remote cache (optional)
+
+### Declaring cache extension
+
+```xml
+
+<extension>
+    <groupId>org.apache.maven.caching</groupId>
+    <artifactId>maven-caching-extension</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+</extension>
+```
+
+### Adding cache config
+
+Copy [default config `maven-cache-config.xml`](maven-cache-config.xml)
```

Comment:
I think generally prefix `maven-` is also redundant because we don't have a `maven-settings.xml`. We have `settings.xml` only.

Reply:
agree, but i would like to limit scope of this pr to documentation only.